packages/core/src/clients/ICacheClient.ts

```diff
@@ -79,6 +80,8 @@ export type SortedSetIncrementOptions = CollectionCallOptions;
 export type SortedSetLengthByScoreOptions = SortedSetLengthByScoreCallOptions;
 
 export interface ICacheClient extends IControlClient, IPingClient {
+  cache(cacheName: string): IMomentoCache;
```

Comment:
This was quite clever. At first I thought why we need to plumb this in `ICacheClient`. With this it's a single new line of code change to get our flavor; without this, we'd have needed changes in constructor initialization as well (Just adding my thought process here; no comment as such)